File: JavaScript/02_cchocolates.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let T;
let testCases = [];

rl.question("", (t) => {
  T = parseInt(t);
  readInput();
});

function readInput() {
  if (testCases.length < T) {
    rl.question("", (input) => {
      const [X, Y, Z] = input.split(" ").map(Number);
      testCases.push({ X, Y, Z });
      readInput();
    });
  } else {
    processInput();
  }
}

function processInput() {
  for (let i = 0; i < T; i++) {
    const { X, Y, Z } = testCases[i];
    const totalRupees = X * 5 + Y * 10;
    const maxChocolates = Math.floor(totalRupees / Z);
    console.log(maxChocolates);
  }
  rl.close();
}

/**********************************************************
 * @INFO
 * Code by Ashish Singh
 * @INFO
 * Gmail - ashishlodhi5559@gmail.com
 * @INFO
 * Github - AshishBytes
 * @INFO
 * LinkedIn - ashishbytes
 * @INFO
 * Instagram - @itz_ash._u
 * @INFO
 * Twitter - @itz_ash_u
 * @INFO
 **********************************************************/

File: Python/02_cchocolates.py
T = int(input())
for _ in range(T):
    X, Y, Z = map(int, input().split())
    total_rupees = X * 5 + Y * 10
    max_chocolates = total_rupees // Z
    print(max_chocolates)

##########################################################
# @INFO
# Code by Ashish Singh
# @INFO
# Github - AshishBytes
# @INFO
# Gmail - ashishlodhi5559@gmail.com
# @INFO
# Instagram - @itz_ash._u
# @INFO
# Twitter - @itz_ash_u
# @INFO
##########################################################
